From 16597563b0088f0361af6576793da04dccab5e7c Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Sat, 2 Oct 2021 00:54:56 -0700
Subject: [PATCH] Consider OE toolchain tuples as armhf

OE toolchains are not having armhf in its name but they still are armhf
ABI, teach this to sgx module build system

Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 .../build/linux2/config/compilers/arm-linux-gnueabi.mk        | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/eurasia_km/eurasiacon/build/linux2/config/compilers/arm-linux-gnueabi.mk b/eurasia_km/eurasiacon/build/linux2/config/compilers/arm-linux-gnueabi.mk
index 0890cb5..7cfd736 100644
--- a/eurasia_km/eurasiacon/build/linux2/config/compilers/arm-linux-gnueabi.mk
+++ b/eurasia_km/eurasiacon/build/linux2/config/compilers/arm-linux-gnueabi.mk
@@ -1,6 +1,6 @@
 # 32-bit ARM soft float compiler
 ifeq ($(MULTIARCH),1)
- TARGET_SECONDARY_ARCH := target_armel
+ TARGET_SECONDARY_ARCH := target_armhf
 else
- TARGET_PRIMARY_ARCH   := target_armel
+ TARGET_PRIMARY_ARCH   := target_armhf
 endif
-- 
2.33.0

